README.md

```diff
@@ -1,54 +1,64 @@
+
+*Remark*: Unfortunately, the "source" spreadsheet neither had presentation dates/times, nor URLs to PDFs. So we might improve upon that (send a PR if you see how).
+
+### :heavy_check_mark: How to Do a Pull Request? Adding Your Poster's Info to the Table
+
+1. <b>GitHub account</b>: Create a <a href="https://github.com">GitHub account</a> if you don’t already have one. As a student, you can apply for the <a href="https://help.github.com/en/github/teaching-and-learning-with-github-education/applying-for-a-student-developer-pack">GitHub Student Developer Pack</a>, which includes offers and benefits from GitHub partners.
+ 2. <b>Forking</b>: Fork https://github.com/datalad-datasets/ohbm2020-posters to create a copy of the repository on your own GitHub account.
+    * Click on `Fork` in the top-right corner of the page and choose to Fork a copy of the repository on your own GitHub account.
+    * You will automatically go to that Fork.<br>
+      <img src="./img/datalad_OHBM2020Posters_icons_2.png" width="700"/>
+ 3. <b>Branching</b>: Within your Fork, create a Branch to add your poster's info to the table.
+    * Click on `Branch: gh-pages` in the semi-top-left corner of the page and write the name of your new Branch (e.g. `poster1929`) in the white rectangle under `Switch branches/tags`. Click on `Enter`.    
+    * You will automatically go to that Branch.<br>
+      <img src="./img/datalad_OHBM2020Posters_4b.png" width="250"/>
+ 4. <b>Editing</b>: Within your Branch, go to the `posters-overrides.json` file & click on the pencil :pencil2: in the top-right corner of the file to edit/add your poster's info (`Video Chat` and `PDF`) to the table.<!--<br>-->
+    <!-- <img src="./img/datalad_OHBM2020Posters_icons_5.png" width="700"/> -->
+    * Search for your poster's number (e.g. `"number": 1929`).
+    * Copy those lines starting with `"videochat"` and `"pdf"` and paste them under your number, e.g. `"number": 1929`. Do not forget to change `JITSI_ROOM_NAME` and `POSTER_PDF_URL` to 1) the name of your own Jitsi room and 2) the URL to your own poster.<br><br>   
+      ```json
+      {
+       "number": 1929,
+       "videochat": "<a href=\"https://meet.jit.si/JITSI_ROOM_NAME\" target=\"JITSI_ROOM_NAME\">jitsi:JITSI_ROOM_NAME</a>",
+       "pdf": "POSTER_PDF_URL"
+      }
+       ```
```

Comment:
It would be nice to add a note that only (and actually any from posters.json) the fields which need  changes should be present here.  So if (as typically) people don't want to change jitsi room -- videochat field better not be copy/pasted

Reply:
Yes indeed, I thought about it, but then I also wanted some feedback. 😄 On it.

Reply:
Reformulated the "Editing" section in [`e56f85d`](https://github.com/sofievdbos/ohbm2020-posters/commit/e56f85d6b02c6c3e441266fb4d0d44939385b7c6). What do you think?